src/ansys/fluent/core/streaming_services/monitor_streaming.py

```diff
@@ -176,7 +176,7 @@ def _process_streaming(self, id, stream_begin_method, started_evt, *args, **kwar
                         if monitor_data:
                             new_df = pd.DataFrame([monitor_data], columns=monitors)
                             new_df.set_index("xvalues", inplace=True)
-                            df_data["df"] = pd.concat([df, new_df])
+                            df_data["df"] = new_df
```

Comment:
Why was this change made?

Reply:
I am trying to resolve this warning:
```
/home/ansys/actions-runner/_work/pyfluent/pyfluent/src/ansys/fluent/core/streaming_services/monitor_streaming.py:179: FutureWarning: The behavior of DataFrame concatenation with empty or all-NA entries is deprecated. In a future version, this will no longer exclude empty or all-NA columns when determining the result dtypes. To retain the old behavior, exclude the relevant entries before the concat operation.
  df_data["df"] = pd.concat([df, new_df])
```
from issue #2117. Is this the incorrect way to change this?

Reply:
Ah yes, for future reference I think it would've probably been more appropriate to tackle the issues in that tracker in a separate PR, rather than bundling them up into this one, as the PR title and description do not really match what you are doing here.

Regarding that warning, if I'm understanding this correctly, would it make sense to check if `df` is indeed empty with `df.empty`, rather than just immediately discarding it? I don't think `df` is always going to be empty (if it is, then that part of the code should probably be revised). Would something like `df = df.append(new_df)` work?

@ajain-work @seanpearsonuk @mkundu1 any advice?

Reply:
Ok I will do that in the future, thank you for the heads up.
When I was looking at it I thought line 167 was creating an empty section to be filled later. I am unsure of how this works as a whole. 
![image](https://github.com/ansys/pyfluent/assets/143635850/2592f504-6134-41bb-900d-7960ae870673)

Reply:
Yeah, in this case it probably is better for a separate PR, but it's fine this time.  You have linked the issue to this PR which is good.